docs/troubleshooting.md

```diff
@@ -188,3 +188,27 @@ Redeploy the broker using origin clients `rollout latest` command.
 $ oc rollout latest asb
 deploymentconfig "asb" rolled out
 ```
+
+### Metrics
+
+The broker will expose [Prometheus](https://prometheus.io/) style metrics that you can use to troubleshoot the broker if none of the steps above work. You can access the published metrics by calling:
```

Comment:
Optional: 
The broker exposes [Prometheus](https://prometheus.io/) style metrics for monitoring and troubleshooting purposes. You can access these metrics via the `/metrics` endpoint by calling:

Reason: It's infinitely useful by itself (as demonstrated already) and isn't conditional on the rest of the troubleshooting steps.